en/advanced/resources.md

```diff
@@ -30,6 +30,8 @@ JabRef allows you to create custom export filters. This functionality and the in
 
 They are collected at [https://layouts.jabref.org](https://layouts.jabref.org).
 
+Styles files are available at [https://layouts.jabref.org](https://layouts.jabref.org) and for LibreOffice at [https://jstyles.jabref.org](https://jstyles.jabref.org). 
```

Comment:
Then, the last paragraph can be deleted?

Reply:
Almost. I did a small modification to keep the reference to jstyle files.